test/unit/storage/FileResourceStore.test.ts

```diff
@@ -205,7 +205,15 @@ describe('A FileResourceStore', (): void => {
     stats.isFile = jest.fn((): any => true);
     (fsPromises.lstat as jest.Mock).mockReturnValueOnce(stats);
     (fs.createReadStream as jest.Mock).mockReturnValueOnce(streamifyArray([ rawData ]));
-    (fs.createReadStream as jest.Mock).mockImplementationOnce((): any => new Error('Metadata file does not exist.'));
+    const readable = streamifyArray([]);
+    readable.on('newListener', (event): void => {
+      if (event === 'open') {
+        setImmediate((): void => {
+          readable.emit('open');
+        });
+      }
+    });
+    (fs.createReadStream as jest.Mock).mockReturnValueOnce(readable);
```

Comment:
Is this actually still necessary now that the new change doesn't listen to the `open` event anymore?

Reply:
nope! will change it